### PR TITLE
ros_comm:691 rosmsg info prints better message if message was not found

### PR DESCRIPTION
**Pull request by NikolasE**
Pull request for https://github.com/ros/ros_comm/issues/691

rosmsg did not show a warning if a msg was not found in the local package (looking for std_msgs/NonExistingMsg already created a message). 

So far no tests, but will be added. Where is the test_ros package that is used in the test cases?

http://answers.ros.org/question/220696/where-can-i-find-the-test_ros-package/

### CODE REVIEW

File: tools/rosmsg/src/rosmsg/__init__.py [hunk 1]
```diff
@@ -419,6 +419,7 @@ def get_msg_text(type_, raw=False, rospack=None):
     :returns: text of .msg file, ``str``
     :raises :exc:`ROSMsgException` If type_ is unknown
     """
+
```
Comment (dirk-thomas) [inline tools/rosmsg/src/rosmsg/__init__.py]: Please don't perform unrelated whitespace changes.

**Comment by dirk-thomas** [comment]
@ros-pull-request-builder

File: tools/rosmsg/src/rosmsg/__init__.py (hunk 1 above)
Comment (NikolasE) [inline tools/rosmsg/src/rosmsg/__init__.py]: sorry, is removed

**Comment by dirk-thomas** [comment]
Thanks for submitting the patch. Closing in favor of #714.
